src/commands/sensor/models/bodies/shareownership-body.ts

```diff
@@ -6,6 +6,9 @@ export class ShareOwnershipBody {
 
   @IsUUID(4)
   @IsNotEmpty()
-  @ApiProperty()
+  @ApiProperty({
+    type: String,
+    description: 'The ownerId of the new co-owner.'
+  })
   readonly ownerId: string;
```

Comment:
Zou dit vanuit de UI ook een lijst kunnen zijn?

Reply:
Het is ook mogelijk tijdens het maken van een sensor dus ook mooi om hier te ondersteunen